### ISSUE TITLE
-(ч)чен

### ISSUE THREAD

**Issue by hectoralos** [apertium/apertium-chv#2]
Suffix/postposition -(ч)чен is currently added always with two чч. Only one should be written after a voiceless consonant:
автанччен, чӗрӗпчен, ҫуртчен, автобусчен, июльччен, пулӑччен, лашаччен, литератураччен.

**Comment by ftyers** [comment]
Analysis:
```
^автанччен/автан<n>+ччен<post>$
^чӗрӗпчен/чӗрӗп<n>+ччен<post>$
^ҫуртчен/ҫурт<n>+ччен<post>$
^автобусчен/автобус<n>+ччен<post>$
^июльччен/июль<n>+ччен<post>$
^пулӑччен/*пулӑччен$
^лашаччен/лаша<n>+ччен<post>$
^литератураччен/литература<n>+ччен<post>$
```
Generation:
```
автан<n>+ччен<post>	автанччен	0,000000
чӗрӗп<n>+ччен<post>	чӗрӗпчен	0,000000
ҫурт<n>+ччен<post>	ҫуртчен	0,000000
автобус<n>+ччен<post>	автобусчен	0,000000
июль<n>+ччен<post>	июльччен	0,000000
пулӑ<n>+ччен<post>	пуллӑччен	0,000000
лаша<n>+ччен<post>	лашаччен	0,000000
литература<n>+ччен<post>	литератураччен	0,000000
```
Is `пулӑччен` correct?

```
$ echo "пулӑ<n>+ччен<post>" | hfst-lookup chv.autogen.hfst 
пулӑ<n>+ччен<post>	пуллӑччен	0,000000
```

I have: 
```
пулӑ:пул%{ː%}ӑ                       N ;          ! ""
```

Mostly fixed in b2b13de .

**Comment by hectoralos** [comment]
I just got a confirmation from Institut Obrazovanija. It seems that this suffix is like others (e.g. instrumental, plural) that "remove" the gemination.
By the way, we still have нефтьччен. Should be нефтьчен.

**Comment by hectoralos** [comment]
Other cases that currently fail:

```
аллея<n>+ччен<post> should be аллейӑччен
министерство<n>+ччен<post> should be министерствӑччен
училище<n>+ччен<post> should be училищӗччен
```

**Comment by jonorthwash** [comment]
@hectoralos, are you completely sure about литератураччен?  @ftyers, are you sure that's the output you were getting?

Currently we have литература behaving just like other Russian nouns with an unstressed word-final а, in that it's spelled with ӑ before most suffixes.  If it should have a stable а per @hectoralos's original message, then the following entry will need to be changed:
```
литература:литератур%{а%}            N ;          ! "" ! литерату́ра
```

I've gotten the remaining examples working per [cv-nouns.yaml](https://github.com/apertium/apertium-chv/blob/master/tests/cv-nouns.yaml).

**Comment by hectoralos** [comment]
@jonorthwash , your definitively right. It is литературӑччен. I was just thinking on ч or чч and nothing else when I was writting the list. I am very sorry.

**Comment by jonorthwash** [comment]
Okay, great, I fixed that.  Are there any other issues with ччен?

**Comment by hectoralos** [comment]
Just пулӑччен instead of  пуллӑччен.

**Comment by jonorthwash** [comment]
> Just пулӑччен instead of пуллӑччен.

Ah, I thought by not responding to Fran's question about that form you were implying that it wasn't a problem.  I'll look at it later if Fran doesn't get to it.

**Comment by ftyers** [comment]
Fixed in 922ff25 .

```
$ echo "пулӑ<n>+ччен<post>" | hfst-lookup chv.autogen.hfst 
пулӑ<n>+ччен<post>	пулӑччен	0,000000
```

**Comment by hectoralos** [comment]
@jonorthwash , my first reply in this post intended to answer Fran's question, but it probably was not clear enough. I'll try to be more concrete. Thanks to both of you for this issue.
